client/src/globalConsts.js

```diff
@@ -41,6 +41,10 @@ const dateFormat = 'en-US';
 
 const temperatureUnits = '°C';
 
+const timeoutToSearchCities = 500;
+
+const startLengthToSearchCitisThroughApi = 2;
```

Comment:
1) Have no idea what is this variable about... 
2) Cities
3) length of what? 
4) are there other ways to search cities? not through (via) API ?